CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 ## Unreleased
 
-* [FEATURE] Add `-kubelet-endpointslice` argument to support `EndpointSlice` for the kubelet controller.
+## 0.77.0 / 2024-09-18
+
+> Along with this release we also introduce a new command line tool [poctl](https://github.com/prometheus-operator/poctl). It is designed specifically for managing Prometheus Operator Custom Resources.
+> This project is currently under active development and fully experimental, so expect breaking changes and rough edges. We encourage you to [try it out](https://github.com/prometheus-operator/poctl/releases/tag/v0.1.0) and provide the feedback.
+
+* [CHANGE] Add more API validation and changes in `ScrapeConfig CRD`. #6921 #6855 #6744
+* [CHANGE] Update the validating webhook to reject `PrometheusRule` objects that would generate configuration greater than the max ConfigMap size (512KiB). #6606
+* [FEATURE] Add experimental Prometheus 3 support. This will allow users to try new Prometheus `v3.0.0-beta.0`. #6940
+  * Feature requests in this version won't be supported till Prometheus 3 is stable but this would let users to try the beta version with Operator and report issues if any.
+* [FEATURE] Allow Prometheus Agent to update, delete, and handle changes to DaemonSet object. #6787
```

Comment:
after more thoughts, I'm not sure that we need to include this in the changelog.